src/psyclone/transformations.py

```diff
@@ -496,6 +497,8 @@ def __init__(self):
         # to the loop directive.
         self._independent = True
```

Comment:
Please could you add/update the tests for this transformation to include using the two new options. The tests for (and in fact the implementation of) this class aren't really in the 'right' place yet but that's not your problem. I think you'll need to update `psyir/transformations/transformations_test.py` and/or `nemo/transformations/openacc/loop_directive_test.py`.

Reply:
I added two new tests in the latter file, basing them off the existing tests. I'd be happy to add tests in the other location, too, but am not exactly sure what's required because the other clauses (`seq`, `independent`) don't seem to be tested there.

Reply:
(See 322dd55fb53be8de9b4885347e94707aafe939ed.)